ui/pages/confirmation/components/confirmation-footer/confirmation-footer.js

```diff
@@ -3,21 +3,23 @@ import PropTypes from 'prop-types';
 import Button from '../../../../components/ui/button';
 
 export default function ConfirmationFooter({
-  onApprove,
+  onSubmit,
   onCancel,
-  approveText,
+  submitText,
   cancelText,
   alerts,
 }) {
   return (
     <div className="confirmation-footer">
       {alerts}
       <div className="confirmation-footer__actions">
-        <Button type="secondary" onClick={onCancel}>
-          {cancelText}
-        </Button>
-        <Button type="primary" onClick={onApprove}>
-          {approveText}
+        {onCancel ? (
+          <Button type="secondary" onClick={onCancel}>
+            {cancelText}
+          </Button>
+        ) : null}
+        <Button type="primary" onClick={onSubmit}>
+          {submitText}
         </Button>
```

Comment:
When there's a single button in the confirmation footer, like in the alert-case, let's remove the `margin-right: 16px `